Resources/ServerInfo/_NF/Guidebook/Rules/7_Antag.xml

```diff
@@ -1,7 +1,37 @@
 <Document>
-# 7. Antagonist Guidelines
+# 7. Antagonism, PvP, and Piracy
 
-    - All ghost roles have a set description for how the role should be played, you must follow the ghost role description unless told otherwise by an admin. Spamming ahelp asking for an exception will likely lead to being banned.
-    - Antagonists are not necessarily your friends. Traitors, rat kings, and sometimes wizards are free agents, but no one should be working together with xenomorphs, nuclear operatives, space dragons, or zombies.
-    - Non-pirates may not assist or work with pirate crews, examples are such as watching mass scanners and tipping them off who to attack, pretending to be hostages, or knowingly funding the pirates for equipment/ships.
-</Document>
\ No newline at end of file
+Even bad guys follow their own rules.
+
+    ## Antagonist Roles
+
+    - When taking a ghost role, follow its set of rules unless told otherwise by an admin.
+    -- Spamming ahelp asking for an exception will likely lead to a ban.
+    - Non-antagonists must not work with antagonists and vice-versa.
+    -- Non-antagonists include xenomorphs, nuclear operatives, space dragons, zombies and pirates.
```

Comment:
Is this supposed to be "Antagonists include" rather than "Non-antagonists include"?